features/link-selectors.yml

```diff
@@ -0,0 +1,15 @@
+name: Link selectors
+description: The `:link` CSS pseudo-class matches unvisited links, `:visited` matches visited links, and `:any-link` matches both.
+spec: https://drafts.csswg.org/selectors-4/#location
+caniuse: css-any-link
+group: selectors
+status:
+  compute_from: css.selectors.any-link
+compat_features:
+  - css.selectors.any-link
```

Comment:
I had no idea this one existed—every day I'm learning about some new-to-me thing on this project. Cool, but slightly concerning.